haiku-apps/sortsave/sortsave-1.1.recipe

```diff
@@ -10,9 +10,7 @@ LICENSE="SortSave"
 REVISION="1"
 SOURCE_URI="https://github.com/bbjimmy/Sort-Save/archive/1.1.tar.gz"
 CHECKSUM_SHA256="cf86c6e650d6313d496a7dce49ebd99377d2b94d6835257881664a30a39ecac6"
-SOURCE_DIR=Sort-Save-1.1
-
-ARCHITECTURES="x86_gcc2 x86 ?x86_64"
+SOURCE_DIR=Sort-Save-1.1ARCHITECTURES="x86_gcc2 x86 ?x86_64"
```

Comment:
I will have to manually go through and fix these most likely because they arise from the SOURCE_DIR not being declared properly in the first place. I will do this in my next PR.